app/validate/hour_validator.py

```
@@ -1,32 +1,40 @@
         input_date = datetime.strptime(now_time, "%Y-%m-%d").date()
         today = datetime.now().date()
         if input_date > today:
-            return  # 미래 날짜는 시간 검증 불필요
+            return  # 미래 날짜는 시간 비교 불필요
         now_time = datetime.now().time()
 
-    # time 객체끼리 비교 (현재 시각과 동일한 슬롯도 과거로 간주)
-    if slot_time <= now_time:
+    now_minutes = now_time.hour * 60 + now_time.minute
+    if slot_minutes <= now_minutes:
         raise PastHourSlotNotAllowedError(f"과거 시간은 허용되지 않습니다: {slot}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)